types/mojang-net/index.d.ts

```diff
@@ -0,0 +1,172 @@
+// Type definitions for Minecraft Bedrock Edition script APIs (experimental) 0.1
+// Project: https://docs.microsoft.com/minecraft/creator/
+// Definitions by: Jake Shirley <https://github.com/JakeShirley>
+//                 Mike Ammerlaan <https://github.com/mammerla>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* *****************************************************************************
+   Copyright (c) Microsoft Corporation.
+   ***************************************************************************** */
+/**
+ * The `mojang-net` module contains types for executing
+ * HTTP-based requests.
+ *
+ * Manifest Details
+ * ```json
+ * {
+ *   // mojang-net
+ *   "uuid": "777b1798-13a6-401c-9cba-0cf17e31a81b",
+ *   "version": [ 0, 1, 0 ]
+ * }
+ * ```
+ *
+ */
+import * as mojangminecraftserveradmin from 'mojang-minecraft-server-admin';
+export enum HttpRequestMethod {
+    /**
+     * Represents the method for an HTTP PUT request. GET requests
```

Comment:
These descriptions for each value are all out of order and some are not internally consistent either. For example, the `POST` value's description is the one for `GET`, but also has "PUT" in the description.